avocado/plugins/config.py

```diff
@@ -50,22 +50,23 @@ def configure(self, parser):
 
     def run(self, config):
         if config.get('config_subcommand') == 'reference':
-            self.handle_reference()
+            self.handle_reference(LOG_UI.debug)
         else:
             self.handle_default(config)
 
-    def handle_reference(self):
+    @staticmethod
+    def handle_reference(print_function):
         full = future_settings.as_full_dict()
         for namespace, option in full.items():
-            LOG_UI.debug(namespace)
-            LOG_UI.debug("~" * len(namespace))
+            print_function(namespace)
+            print_function("~" * len(namespace))
             help_lines = textwrap.wrap(option.get('help'))
             for line in help_lines:
-                LOG_UI.debug(line)
-            LOG_UI.debug("")
-            LOG_UI.debug("* Default: %s", option.get('default'))
-            LOG_UI.debug("* Type: %s", option.get('type'))
-            LOG_UI.debug("")
+                print_function(line)
+            print_function("")
+            print_function("* Default: %s" % option.get('default'))
+            print_function("* Type: %s" % option.get('type'))
+            print_function("")
```

Comment:
I didn't get why this change, but I'm ok with that.

Reply:
Do you mean this patch, or this block? I'm curious.

Reply:
Adding a `print_function` argument and replacing instead of using directly the LOG_UI. I'm assuming that is to make it easy to replace the log level at one place instead of multiple lines.

Reply:
Because we need to output to a file isntead here:

https://github.com/avocado-framework/avocado/pull/3946/files#diff-1ec61223727a10e5e4c74c68336809fcR40

Reply:
@clebergnu got it. thanks. Need more coffee here.

Reply:
> @clebergnu got it. thanks. Need more coffee here.

:coffee: